dftimewolf/lib/processors/workspace_audit_timesketch.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+"""Processes Google Workspace logs for loading into Timesketch."""
+
+import os
+import tempfile
+import json
+import string
+
+from dftimewolf.lib.module import BaseModule
+from dftimewolf.lib.containers import containers
+
+from dftimewolf.lib.modules import manager as modules_manager
+
+
+class WorkspaceAuditTimesketch(BaseModule):
+  """Transforms Google Workspace logs for Timesketch."""
+
+  _FORMAT_STRINGS_PATH = os.path.join(os.path.dirname(__file__),
+      'workspace_format_strings.json')
+
+  def __init__(self, state, name=None, critical=False):
+    super(WorkspaceAuditTimesketch, self).__init__(
+        state, name=name, critical=critical)
+
+    with open(self._FORMAT_STRINGS_PATH, 'r') as formatters_json:
+      self._all_application_format_strings = json.load(formatters_json)
+
+  def SetUp(self, *args, **kwargs):
+    """Sets up necessary module configuration options."""
+    # No configuration required.
+
+  def _FlattenActor(self, actor_dict):
+    """Flattens out actor information from a Workspace log record.
```

Comment:
It looks more like you're returning a subset of the keys in `actor_dict` rather than flattening it. Is this really flattening. Wouldn't `_ExtractActorInfo` be a better name for this function?

Reply:
Done